app/src/lib/editors/win32.ts

```diff
@@ -161,12 +161,24 @@ const editors: IWindowsExternalEditor[] = [
   },
   {
     name: 'Sublime Text',
-    registryKeys: [LocalMachineUninstallKey('Sublime Text 3_is1')],
+    registryKeys: [
+      // Sublime Text version 3
+      LocalMachineUninstallKey('Sublime Text 3_is1'),
+      // Sublime Text version 4 (private beta)
+      LocalMachineUninstallKey('Sublime Text_is1'),
+    ],
```

Comment:
It would appear that the order here signifies priority. With the [official release](https://www.sublimetext.com/blog/articles/sublime-text-4) of Sublime Text 4 wouldn't it make sense to prioritize 4 over that of 3?